src/tests/utils/fixtures/endpoint_request_fixture.py

```diff
@@ -10,7 +20,60 @@ def __init__(self, url_rule=None, method=None, headers=None):
 def create_request(url, method, access_token):
 
     request = Request(
-        url_rule=url, method=method, headers={"Authorization": "access_token " + access_token}
+        url_rule=url,
+        method=method,
+        headers={"Authorization": "access_token " + access_token},
+    )
+
+    return request
+
+
+def create_refresh_token_request(
```

Comment:
What that mean create refresh token request? this does not look like a request, looks like something different not related to requests, could you give it a better name?

Also if there are other functions with name please update them

Reply:
Why this is creating a request and inserting data in the data base? please never do that, try to refactor this, these are different operations

Reply:
Done.